### DISCUSSION TITLE
PP being calculated

### DISCUSSION THREAD

**Discussion by zer0stamina** [ppy/osu#25642] [Q&A] [answered]
I contacted osu support about this and was told to make a discussion post here, so here I am.

I achieved a #1 score on a beatmap, nothing impressive (https://lazer.ppy.sh/beatmapsets/78164#osu/221888) , but the PP has been calculating for months now. Was advised this may be because I had increased the speed of double time.

**Comment by Joehuu** [comment, marked as answer]
> Was advised this may be because I had increased the speed of double time.

Your suspicion is true, see the code comment in https://github.com/ppy/osu-queue-score-statistics/blob/eebd505535b1a2a8f8780b676acb941a29b5c06f/osu.Server.Queues.ScoreStatisticsProcessor/Processors/ScorePerformanceProcessor.cs#L174.
